config/pagination.php

```diff
@@ -0,0 +1,15 @@
+<?php
+return [
+
+    /*
+
+    |--------------------------------------------------------------------------
+    | PAGINATION
+    |--------------------------------------------------------------------------
+    |
+    */
+
+
+
+    'six' => '6',
```

Comment:
Don't try to make a config with the same mirror.
Make file `config/const.php`
Add value in file look like.
```
<?php
return [
    'limit_record' => 6,
];

```